src/main/java/model/figure/Circle.java

```diff
@@ -0,0 +1,23 @@
+package model.figure;
+
+import model.Figure;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public int calculateArea() {
+        return (int) (Math.PI * (radius * radius));
+    }
```

Comment:
The method name 'calculateArea' is a verb, which is good practice for naming methods.